blazor/pdfviewer-2/how-to/deploy-maui-using-android-emulator.md

```diff
@@ -60,11 +60,15 @@ To run the PDF Viewer (Next Gen) in a Blazor Android MAUI application using the
 
 ![android-sdk-liscence](../getting-started/gettingstarted-images/android-sdk-liscence_maui.png)
 
-**Step 2** Install and launch the Android Device Manager. Open the Android SDK Manager, go to the `SDK Tools` tab, select the `Android Device Manager` checkbox, and click `Apply` or `OK`. This will allow you to create, manage, and launch Android Virtual Devices (AVD's) for testing and running Android applications.
+**Step 2** Right-click the **~/Pages/Index.razor** file in Solution Explorer, go to `Properties`, and set Build Action to `Embedded Resource` and Copy to Output Directory to `Copy Always`.
```

Comment:
I thought it is not needed. May be this step needed for .pdf file not for this razor file.